elfi/methods/bo/utils.py

```diff
@@ -162,3 +162,67 @@ def evaluate_gradient(self, x):
         x = np.atleast_2d(x)
         n, input_dim = x.shape
         return self.scale * self.gradient(x).reshape(n, input_dim)
+
+
+def make_additive_acq(acquisition_class, function):
+    """Make acquisition function adjusted with an additive term.
+
+    Parameters
+    ----------
+    acquisition_class : Type[elfi.methods.bo.acquisition.AcquisitionBase]
+        Acquisition function to be adjusted.
+    function : AdjustmentFunction
+        Function added to the base acquisition function.
+
+    Returns
+    -------
+    AdditiveAcquisition
```

Comment:
btw i realised this should be `Type[AdditiveAcquisition]` since the function returns the class (not an instance). i will fix this